Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/SingleZone/VAV/SetPoints/Supply.mo

```diff
@@ -527,6 +549,11 @@ based on the same temperature sensors and control loops.
 </html>", revisions="<html>
```

Comment:
This block uses a figure Resources/Images/Controls/OBC/ASHRAE/G36_PR1/AHUs/VAVSingleZoneTSupSet.png that does not exist. Please correct, and also make sure that `modelica-json` detects such cases.